content/git/concepts/revert/revert.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: 'Revert'
+Description: 'The git revert command is used to create a new commit that will undo any changes made to a previous commit.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the **`revert`** command is used to create a new commit that will undo any changes made to a previous commit. This can be considered as the safe undo command because it keeps previous history. The command is best used when working with others and need to preserve the commit history.
```

Comment:
```suggestion
In Git, the **`revert`** command is used to create a new commit that will undo any changes made to a previous commit. This can be considered as the safe undo command because it keeps previous history. The command is best used when working with others and there's a need to preserve the commit history.
```